### PR TITLE
[ZH] Twitter

### PR DESCRIPTION
**Pull request by kj415j45**
Reviewed in osu-translate-zh#78

---

### CODE REVIEW

**Comment by Joehuu** [comment]
Please see the English file. Update accordingly.

**Comment by kj415j45** [comment]
Fixed, thanks

File: wiki/Twitter/zh.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+## 我们也发推文！
+
+osu! 有数个 Twitter 账号, 每个服务都有其明确的目的。 大多数访问量都很少，但是会帮助你了解 osu! 的游戏系统。
+
+| 头像 | 账号 | 描述 |
+| :-: | :-: | --- |
+| ![osu! 头像](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | 消息和通知的官方源，没人看但内容质量高。 |
+| ![osu!status 头像](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | 致力于提供网站那些不被人关注的通知和 bancho 的问题。如果你只想关注一个账号，这就是你需要的。 |
+| ![Dean Herbert 头像](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | 不只限于 osu! 相关, 但会我们会把这里留给那些感兴趣的人。 ppy的生活毕竟大部分还是与 osu! 有关。 |
+| ![osu!support 头像](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | 帮助处理账号/社区的问题。 |
```
Comment (Joehuu) [inline wiki/Twitter/zh.md]: `osu!support` -> `osu! support`

**Comment by Joehuu** [comment]
Can someone do this simple change? It's good after that. Also fix the description by adding a divider and optionally add a subtext.

**Comment by Joehuu** [comment]
Bumping this easy change and merge. Deleting this comment.

**Pull request merged**